girder/api/v1/file.py

```diff
@@ -109,16 +109,38 @@ def initUpload(self, parentType, parentId, name, size, mimeType, linkUrl, refere
                 self.requireAdmin(
                     user, message='You must be an admin to select a destination assetstore.')
                 assetstore = self.model('assetstore').load(assetstoreId)
+
+            chunk = None
+            if 'chunk' in params:
+                chunk = params['chunk']
+                if isinstance(chunk, cherrypy._cpreqbody.Part):
```

Comment:
I've added a note the `POST /file` and `POST /file/chunk` with some documentation.